### DISCUSSION TITLE
custom page size v2

### DISCUSSION THREAD

**Discussion by aditfurk** [diegomura/react-pdf#2008] [Q&A]
Hi, can anyone tell me how to create custom page size? 
i have tried 

<Page size={{ width: "8cm", height: "10cm" }} /> not working,

i also have tried copied array from the default page size

<Page size={[595.28, 841.89]} /> // the default number for A4
 and its working, but the problem is how can i convert from cm or mm to those array of numbers
 
 Thanks...

**Comment by mybelatar** [comment]
did you find a solution ?

**Comment by nikischin** [comment]
The problem seems to be the issue implemented with #1869. Generally speaking internally the numbers are basically a 72 dpi representation of the measurements. so `[8cm, 10cm]` could be represented as `[227, 283]`. However, you shouldn't change the dpi setting as this could result in weird effects.

**Reply by nikischin** [reply]
I opened a pr #2773 which does solve your problem 😄

**Reply by eduardo-softeasy** [reply]
Hi guys, is this already working?

**Reply by nikischin** [reply]
> Hi guys, is this already working?

Hi @eduardo-softeasy unfortunately the PR is not yet merged.
